api/document/js/commands.ts

```diff
@@ -69,3 +70,46 @@ export function makeSaveThenXml(api: Api) {
     window.location.assign(`${window.location.href}/akn`);
   };
 }
+
+const inLi = (pos: ResolvedPos) => (
+  pos.depth >= 3
+  && pos.node(pos.depth).type === schema.nodes.inline
+  && pos.node(pos.depth - 1).type === schema.nodes.para
+  && pos.node(pos.depth - 2).type === schema.nodes.listitem
+  && pos.node(pos.depth - 3).type === schema.nodes.list
+);
```

Comment:
This makes me wish we had a sort of "CSS selectors, but for ProseMirror document schema".

Reply:
Agreed. The `pathToResolvedPos()` function implements some of that, but it's pretty rudimentary.